packages/gateway/src/features/registration/type-resolvers.ts

```diff
@@ -505,7 +505,7 @@ export const typeResolvers: GQLResolver = {
       await getCertificatesFromTask(task, _, authHeader),
     assignment: async (task, _, authHeader) => {
       const assignmentExtension = findExtension(
-        `${OPENCRVS_SPECIFICATION_URL}extension/regAssigned`,
+        `${OPENCRVS_SPECIFICATION_URL}extension/regLastUser`,
```

Comment:
@CornelMaze What we should be doing here is that, look for the `regAssigned` extension, if found (which we were doing previously) then find the `regLastUser` and use that data